src/tomahawk/TomahawkWindow.cpp

```diff
@@ -757,11 +757,23 @@ TomahawkWindow::changeEvent( QEvent* e )
 }
 
 
+// CHANGED
 void
 TomahawkWindow::closeEvent( QCloseEvent* e )
 {
+
+    // CHANGED change according to definitions
+    TomahawkSettings* s = TomahawkSettings::instance();
+    //QMainWindow::setQuitOnLastWindowClosed(s->exitOnClose());
```

Comment:
Why is this code commented out? If it doesn't work don't add it at all, or at least leave a comment why it's commented out